ecl/hql/hqlrepository.cpp

```diff
@@ -26,6 +26,36 @@
 #include "hqlutil.hpp"
 #include "jsecrets.hpp"
 
+static std::atomic<unsigned> fetchNesting;
+static std::atomic<bool> abortPending;
```

Comment:
hesitated that these were auto-initialized to 0, clearer if explicitly {0} / {false} ?

Reply:
statics are always zero initialised, but no harm in making it explicit.